pages/filterCards/index.tsx

```diff
@@ -0,0 +1,61 @@
+import axios from 'axios';
+import { withRouter } from 'next/router';
+import * as React from 'react';
+
+interface IProps {
+  data: {
+    tags: any;
+  };
+}
+
+class Index extends React.Component<IProps> {
+  public static async getInitialProps() {
+    const res = await axios.get('http://localhost:8080/api/cards/filter');
```

Comment:
I'm a little blocked by the task how to get current `url` of the page..

Reply:
`window.location.host` ? or what do you mean?

Reply:
It works only on the client side but here we render this component at server and we don't have `window`.

Reply:
`req.hostname` ? http://expressjs.com/en/api.html

Reply:
better move it to config